crates/routes/src/feeds.rs

```diff
@@ -38,6 +38,8 @@ const RSS_FETCH_LIMIT: i64 = 20;
 #[derive(Deserialize)]
 struct Params {
   sort: Option<String>,
+  limit: Option<i64>,
+  after: Option<i64>,
```

Comment:
Don't use after. Use limit and page like all the other objects.

Reply:
Cool, reverted back to using page